packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -26,10 +26,10 @@ const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
                     onClick={() => sendbird_store.setShouldShowChatModal(true)}
                 />
             )}
-            renderPageFooterChildren={() =>
-                order_store.order_information.should_show_order_footer && (
-                    <OrderDetailsFooter order_information={order_store.order_information} />
-                )
+            renderPageFooterChildren={
+                order_store.order_information.should_show_order_footer
+                    ? () => <OrderDetailsFooter order_information={order_store.order_information} />
+                    : null
```

Comment:
Why do we have to change this to a ternary?

Reply:
The previous change uses my created function `shouldShowOrderDetailsFooter`, which has to be placed outside of the parent component `OrderDetailsWrapper` in order to avoid mounting-remounting: https://levelup.gitconnected.com/code-review-avoid-declaring-react-component-inside-parent-component-1768a645f523
However since this expression is not reused anywhere else in the code and the ternary expression is not a deeply nested and complex one, there would be no need to declare a function and pass in multiple arguments outside the parent component